api/types/oidc.go

```diff
@@ -88,6 +88,10 @@ type OIDCConnector interface {
 	// GetGoogleAdminEmail returns a google admin user email
 	// https://developers.google.com/identity/protocols/OAuth2ServiceAccount#delegatingauthority
 	// "Note: Although you can use service accounts in applications that run from a G Suite domain, service accounts are not members of your G Suite account and aren’t subject to domain policies set by G Suite administrators. For example, a policy set in the G Suite admin console to restrict the ability of G Suite end users to share documents outside of the domain would not apply to service accounts."
+	GetGoogleServiceAccount() string
+	// GetGoogleServiceAccount retrieves service account json for Google
```

Comment:
This comment should be above the `GetGoogleServiceAccount()` line

Reply:
moved